files/producer.py

```diff
@@ -56,19 +57,19 @@ def process_condor_status_output(condor_status_output: str) -> str:
     return processed_output
 
 
-def get_condor_status(cluster_status_script_file: str) -> str:
+def get_condor_status() -> list:
     """
-    Get condor status from shell script output
+    Get condor metrics from scheduler python bindings
     """
-    condor_metrics = process_condor_status_output(subprocess.check_output(["sh", cluster_status_script_file]).decode("utf-8").strip())
 
-    # Add timestamp
-    now = time.time()
-    condor_metrics = f"{condor_metrics},querytime={now}"
+    from htcondor import collect_metrics
```

Comment:
I think we should change the name of the script to `htcondor_cluster_util` or something because there is also another Python package named `htcondor` which will be installed when installing HTCondor, so to avoid any confusion and issues, it would be nice to change it.